questions/qa-bidi-unicode-controls.en.html

```diff
@@ -164,138 +178,230 @@ <h2>Changing the base direction</h2>
           <td>&lt;/bdo&gt;</td>
           <td>used for RLO or LRO</td>
         </tr>
-        <tr>
-          <td>PDI</td>
-          <td>POP DIRECTIONAL ISOLATE</td>
-          <td>U+2069</td>
-          <td>end tag</td>
-          <td>used for RLI, LRI or FSI</td>
-        </tr>
       </tbody>
     </table>
     <p>These characters are invisible, although in some editors it may be possible to show symbols that represent them. You could also use character escapes to represent them, such as <code>&amp;#x2067;</code>, but in bidirectional source text you may find that the characters in the escape don't stay together. (See <a href="https://www.w3.org/TR/2014/NOTE-i18n-html-tech-bidi-20140603/#bidisource"><cite>Problems with bidirectional source text in markup</cite></a> for more on this.)</p>
```

Comment:
I think we have deprecated this document. It might be better not to link to it.

Reply:
Mm. I thought we have already copied that section to another document, but i'm not seeing it.  So i'm going to port the information into a new article, and then we can point to that.

Reply:
This now links to the future location of the bidi-source article.